.gitignore

```diff
@@ -106,3 +106,8 @@ src/sous_chef/static
 
 # Ignore generated pdf files.
 src/*.pdf
+
+# the magic docker-compose override file is for overriding settings locally
+# it should not be a part of the repo. If you want per-environment settings,
+# use an explicit `docker-compose -f docker-compose.yml -f docker-compose.environment.yml`
+docker-compose.override.yml
```

Comment:
Thanks for the link. Having the laser focus there really helps move the debate forward, and I agree that you're doing it just as they suggest.

I still think it's wrong. `docker-compose.yml` can have dev settings in it by default which are then overridden by `docker-compose.prod.yml`. "override" says to me "these are settings which are custom to this specific instance, which I need in order"; if they meant "override" to always mean "dev" then they should have called it "dev". Emphatically I think that name means they must not be stored as part of the codebase, and not be in the repo, be `.gitignore`'d.